build.gradle

```diff
@@ -277,12 +284,29 @@ dependencies {
 
   compile "ch.qos.logback:logback-classic:${logbackVersion}"
 
-  testImplementation "net.serenity-bdd:serenity-core:${serenityCoreVersion}",
-          "net.serenity-bdd:serenity-junit:${serenityCoreVersion}",
-          "net.serenity-bdd:serenity-screenplay:${serenityCoreVersion}",
-          "junit:junit:${junitVersion}",
-          "org.junit.vintage:junit-vintage-engine:5.6.2",
-          "org.assertj:assertj-core:${assertJVersion}"
+  testImplementation group: 'org.springframework.boot', name: 'spring-boot-starter', {
+    exclude group: 'org.hamcrest', module: 'hamcrest-core'
+  }
+
+  testCompile group: 'org.springframework.boot', name: 'spring-boot-starter', {
+    exclude group: 'org.hamcrest', module: 'hamcrest-core'
+  }
+  testCompile ("org.springframework.boot:spring-boot-starter-test")
```

Comment:
are we using all these libraries? if not remove unused ones

Reply:
Yes